Backend/Remora.Discord.Rest/API/Channels/DiscordRestChannelAPI.cs

```diff
@@ -109,6 +126,7 @@ public virtual Task<Result<IChannel>> GetChannelAsync
         Optional<IReadOnlyList<Snowflake>> appliedTags = default,
         Optional<SortOrder> defaultSortOrder = default,
         Optional<ForumLayout> defaultForumLayout = default,
+        Optional<string?> status = default,
```

Comment:
`ModifyChannelAsync` should not be modified as it is not meant to be used for setting the voice channel status by documentation

Reply:
I swear I removed that. Bwah

Reply:
dw, there are many Modify...Channel calls